File: C# OOP/Exams/C# OOP Exam - 16 August 2020/01. Structure_Skeleton/OnlineShop-Skeleton/OnlineShop/Models/Products/Computers/DesktopComputer.cs

namespace OnlineShop.Models.Products.Computers
{
    public class DesktopComputer : Computer
    {
        private const int DESKTOP_OA_PERFORMANCE = 15;
        public DesktopComputer(int id, string manufacturer, string model, decimal price)
            : base(id, manufacturer, model, price, DESKTOP_OA_PERFORMANCE)
        {
        }
    }
}

File: C# OOP/Exams/Java OOP Exam 16 August 2020/OnlineShop/src/onlineShop/models/products/computers/DesktopComputer.java
package onlineShop.models.products.computers;

public class DesktopComputer extends BaseComputer{

    private static final int DEF_DESKTOP_PERFORMANCE = 15;
    public DesktopComputer(int id, String manufacturer, String model, double price) {
        super(id, manufacturer, model, price, DEF_DESKTOP_PERFORMANCE);
    }
}
